modules/sdk-coin-atom/src/lib/iface.ts

```diff
@@ -0,0 +1,35 @@
+import { TransactionExplanation as BaseTransactionExplanation, TransactionType } from '@bitgo/sdk-core';
+import { Coin, SignerData } from '@cosmjs/stargate';
+
+export interface TransactionExplanation extends BaseTransactionExplanation {
+  type: TransactionType;
+}
+
+export interface MessageData {
+  typeUrl: string;
+  value: {
+    fromAddress: string;
+    toAddress: string;
+    amount: Coin[];
+  };
+}
+
+export interface GasFeeLimitData {
+  amount: Coin[];
+  gas: number; // gas limit
```

Comment:
Based on our sandbox scripts, `gasLimit` seems like a `string`

Reply:
I think it might be getting converted in their SDK, because in my debugging when deserializing from an encoded base64 string, it came back as a number.

Reply:
if it came back as long then BigInt would be better option I think. JS number type is notorious for precision issues